cmd/erasure-readfile.go

```diff
@@ -193,8 +199,9 @@ func (s ErasureStorage) ReadFile(writer io.Writer, volume, path string, offset,
 		buffOffset += chunksize
 
 		if needsReconstruction {
-			if err = s.ErasureDecodeDataBlocks(blocks); err != nil {
-				return f, errors.Trace(err)
+			if err = s.ErasureDecodeDataBlocks(ctx, blocks); err != nil {
+				logger.LogIf(ctx, err)
```

Comment:
It looks like there is no need to log anything here since `ctx` is passed to `s.ErasureDecodeDataBlocks` method and the logging is handled in there.

Reply:
done